influxdb_connector/influxdb_connector.py

```diff
@@ -53,34 +101,34 @@ def influxdb_query(url, token, org, bucket, range_start):
     """
     client = influxdb_client.InfluxDBClient(url=url, token=token, org=org)
     query_api = client.query_api()
-    if range_start.lower() != 'none':
-        query = 'from(bucket: "' + bucket + '")\n |> range(start:' + range_start + ')'
+    if range_start.lower() != "none":
+        query = 'from(bucket: "' + bucket + '")\n |> range(start:' + range_start + ")"
     else:
         query = 'from(bucket: "' + bucket + '")\n |> range(start:-10y)'
     tables = query_api.query(query, org=org)
     csv_builder = dict()
-    csv_builder['time'] = []
+    csv_builder["time"] = []
 
     # build dictionary from query
     for table in tables:
```

Comment:
Recommend adding comments for readability

Reply:
Added comments on influxdb_query function for better readability